notebooks/290-magika-content-type-recognition/290-magika-content-type-recognition.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>Add please Table of Contents</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1826/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>